_dashboards/admin-ui-index/sm-dashboards.md

```diff
@@ -146,4 +165,11 @@ Use the steps below to take a snapshot manually:
 
  After the restore operation is complete, the restored indexes are listed in the **Indices** panel. To view the indexes, in the left panel, under **Index Management**, select **Indices**.
 
-<img src="{{site.url}}{{site.baseurl}}/images/restore-snapshot/restore-snapshot-indices-panel.png" alt="View Indices">{: .img-fluid}
\ No newline at end of file
+<img src="{{site.url}}{{site.baseurl}}/images/restore-snapshot/restore-snapshot-indices-panel.png" alt="View Indices">{: .img-fluid}
+
+#### Delete a snapshot
```

Comment:
Is this section supposed to be after lines 174 and 175? Also, is it supposed to be heading level 2?